### DISCUSSION TITLE
Add a "copy highlighted text" icon to the popup?

### DISCUSSION THREAD

**Discussion by savagestranger** [YoshifumiFuyuno/Search-from-Popup-or-ContextMenu#88] [Q&A]
In the Firefox addon, there was an icon to simply copy the text that was highlighted.  Is there an option for this that I'm overlooking?
Thanks!

**Comment by YoshifumiFuyuno** [comment]
Hi.

Sorry for the late reply.
The engine for copying selections is below.
https://github.com/YoshifumiFuyuno/Search-from-Popup-or-ContextMenu/wiki/Example-engines#example-engines
* copyToClipboardAsText
* copyToClipboardAsHtml

Just click to register.

Best regards
